modules/ROOT/pages/maintenance/b-r/backup_considerations.adoc

```diff
@@ -0,0 +1,271 @@
+
+[.gray-light-background,subs="macros"]
+----
+    ~/.ocis/proxy/:tree
+.
+├── server.crt
+└── server.key
+
+1 directory, 2 files
+----
+
+* `server.crt`: +
+*BACKUP OPTIONAL*. This is the certificate for the http services. Use `PROXY_TRANSPORT_TLS_CERT` to specify its path.
+* `server.key`: +
+*BACKUP OPTIONAL*. This is the certificate key for the http services. Use `PROXY_TRANSPORT_TLS_KEY` to specify its path.
+
+=== search
+
+Contains the index created by search. See the xref:{s-path}/search.adoc[SEARCH service] for more details.
+
+[.gray-light-background,subs="macros"]
+----
+    ~/.ocis/search/:tree -L 1
+.
+└── bleve
+
+2 directories, 0 files
+----
+
+* `bleve`: +
+*BACKUP RECOMMENDED/OPTIONAL*. This contains the search index. Can be specified via `SEARCH_ENGINE_BLEVE_DATA_PATH`. If not backed up, the search index needs to be recreated. This can take a long time depending on the amount of files.
+
+=== storage
+
+Contains Infinite Scale meta (and blob) data, depending on the blobstore. See the xref:{s-path}/storage-users.adoc[STORAGE-USERS service] for more details.
+
+[.gray-light-background,subs="macros"]
+----
+    ~/.ocis/storage/:tree -L 1
+.
+├── metadata
+├── ocm
+└── users
+
+4 directories, 0 files
+----
+
+* `metadata`: +
+*BACKUP REQUIRED*. Contains system data. Path can be specified via `STORAGE_SYSTEM_OCIS_ROOT`. Not backing it up will remove shares from the system and will also remove custom settings.
+* `ocm`: +
+*BACKUP REQUIRED/OMITABLE*. Contains ocm share data. When not using xref:{s-path}/ocm.adoc[ocm sharing], this folder does not need to be backed up.
+* `users`: +
+*BACKUP REQUIRED*. Contains user data. Path can be specified via `STORAGE_USERS_OCIS_ROOT`. Not backing it up will remove all spaces and all files. As result, you will have a configured but empty Infinite Scale instance, which is fully functional accepting new data. Old data is lost.
+
+=== thumbnails
+
+Contains thumbnails data. See the xref:{s-path}/thumbnails.adoc[THUMBNAILS service] for more details.
+
+[.gray-light-background,subs="macros"]
+----
+    ~/.ocis/thumbnails/:tree -L 1
+.
+└── files
+----
+
+* `files`: +
+*OPTIONAL/RECOMMENDED*. This folder contains prerendered thumbnails. Can be specified via `THUMBNAILS_FILESYSTEMSTORAGE_ROOT`. If not backed up, thumbnails will be regenerated automatically on access which leads to some load on the thumbnails service.
+
+=== web
+
+Contains web assets such as custom logos, themes etc. See the xref:{s-path}/web.adoc[WEB service] for more details.
+
+[.gray-light-background,subs="macros"]
+----
+    ~/.ocis/web/:tree -L 1
+.
+└── assets
+
+2 directories, 0 files
+----
+
+* `assets`: +
+*BACKUP RECOMMENDED/OMITABLE*. This folder contains custom web assets. Can be specified via `WEB_ASSET_CORE_PATH`. If no custom web assets are used, there is no need for a backup. If those exist but are not backed up, they need to be reuploaded.
```

Comment:
```suggestion
*BACKUP RECOMMENDED/OMITTABLE*. This folder contains custom web assets. Can be specified via `WEB_ASSET_CORE_PATH`. If no custom web assets are used, there is no need for a backup. If those exist but are not backed up, they need to be reuploaded.
```

I think OMITTABLE is the correct spelling, but my spell checker does not like the word at all. Online dictionary resources have OMITTABLE.